### DISCUSSION TITLE
Links to SignServer documentation appear to be broken?

### DISCUSSION THREAD

**Discussion by rws-githubuser1** [Keyfactor/signserver-ce#59] [Q&A] [answered]
I'm attempting to access to SignServer documentation on the https://www.signserver.org/ site, but all the links that point to https://doc.primekey.com/signserver (or its sub-directories) fail. Is anyone else having the same experience? I'd appreciate any help getting access to SignServer documentation. Thanks.

**Comment by netmackan** [comment, marked as answer]
Unfortunately the doc site is currently taken down for maintainency. I hope it will be back soon.
Note that an export if the documentation is also available in the release zip-files. 'After extracting them it is available in doc/htdocs/index.html which can be opened in a web browser.

Cheers,
Markus

**Reply by rws-githubuser1** [reply]
Hi Markus, thanks for your helpful feedback :)  I'll check the Zip.
